skuld/openstackmap.py

```diff
@@ -312,7 +316,7 @@ def _get_keystone_data(self):
                     f:
                 pickle.dump(tenants_by_name, f)
 
-            with open(self.pers_keystone + '/asignments.pickle', 'wb') as f:
+            with open(self.pers_keystone + '/roles_a.pickle', 'wb') as f:
                 pickle.dump(roles_a, f, protocol=-1)
```

Comment:
No, it is the opposite. A negative number is the parameter to pass to pickle to ask it using the last version of the protocol, instead of a specific one according to its documentation.

By the way, that line is not from this pull request ;-)